.gitignore

```diff
@@ -86,6 +86,7 @@ celerybeat-schedule
 .venv
 venv/
 ENV/
+virtualenv/
```

Comment:
Please remove this change. You should add things that are specific to your local setup to the [global gitignore](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).